archivers/libaec/Portfile

```diff
@@ -12,17 +19,14 @@ description         Adaptive Entropy Coding library
 long_description \
     Libaec provides fast lossless compression of 1 up to 32 bit wide\
     signed or unsigned integers (samples). The library achieves best\
-    results for low entropy data as often encountered in space imaging\
+    results for low entropy data, as often encountered in space imaging\
     instrument data or numerical model output from weather or climate\
     simulations. While floating point representations are not directly\
     supported, they can also be efficiently coded by grouping exponents\
-    and mantissa.
-
-homepage            https://gitlab.dkrz.de/k202009/libaec
-master_sites        ${homepage}/uploads/ea0b7d197a950b0c110da8dfdecbb71f
-
-checksums           rmd160  e41987d68a1bd5325c9016bb161b51a2b845f81a \
-                    sha256  f2b1b232083bd8beaf8a54a024225de3dd72a673a9bcdf8c3ba96c39483f4309 \
-                    size    3120061
+    and mantissa.\
+    \
+    Libaec includes a free drop-in replacement for the SZIP\
+    library. Just replace SZIP's shared library libsz.so* with\
+    libaec.so* and libsz.so* from libaec.
 
-configure.args      --prefix=${prefix}/lib/${name}
+configure.pre_args  -DCMAKE_INSTALL_PREFIX=${prefix}/lib/${name}
```

Comment:
this should probably be "configure.args-append" as it is in almost every other Portfile?

Reply:
My mistake, will fix.  This is my first cmake port, and I just copied that from some other similar port.

Reply:
Are you sure this prefix setting is needed?  It's resulting in seemingly incorrect install destinations:

```
/opt/local/lib/libaec/bin/aec
/opt/local/lib/libaec/share/man/man1/aec.1
/opt/local/lib/libaec/lib/libaec.0.0.12.dylib
```

This does not look right to me. I think you can delete this line so that CMake will use MacPorts' default prefix.

Reply:
True, that's indeed just wrong besides the syntax 😉

Reply:
@reneeotten, what was that about the syntax?  Was that "configure.args-append", or something else?